octodns/cmds/report.py

```diff
@@ -41,24 +57,40 @@ def main():
         help='Source(s) to pull data from',
     )
     parser.add_argument(
-        '--num-workers', default=4, help='Number of background workers'
+        '--concurrency',
+        type=int,
+        default=10,
```

Comment:
The default is purposefully low as it's pretty easy to get rate limited/blocked by public DNS resolvers like Google and Cloudflare when running reports for large zones against them. 

```suggestion
        default=4,
```